src/redis-cli.c

```diff
@@ -6402,6 +6414,14 @@ static int clusterManagerCommandImport(int argc, char **argv) {
                 src_port, src_ctx->errstr);
         goto cleanup;
     }
+    //Auth for the source node. 
```

Comment:
```suggestion
    /* Auth for the source node. */
```

We don't use the `//` in this codebase, and anyway there was a missing space :)

Reply:
@itamarhaber thanks Itamar, however I found the in this function is using // as function line comments, that's why I use // just to be consistant. maybe it was edited by different person.. please see https://github.com/redis/redis/blob/unstable/src/redis-cli.c#L6329 and L6359..

Reply:
Understood - so for consistency keep it but for correctness(?) just add the missing space. I guess someone should convert sometime all `//` to `/* ... */` to be consistent with the unwritten code style guide.

Reply:
@itamarhaber agree, maybe we also need a similar approach as the typo one for these coding style fix. But before that we may need to put our coding style guide somewhere for contributors to reference.